packages/suite/src/actions/wallet/selectedAccountActions.ts

```diff
@@ -147,7 +149,11 @@ const getAccountLoader = (
 };
 
 // display notification above the component, with or without component body
-const getAccountNotification = (state: AppState, selectedAccount: SelectedAccountState) => {
+const getAccountNotification = (
+    state: AppState,
+    selectedAccount: SelectedAccountState,
+    dispatch: Dispatch,
```

Comment:
just a thought... since we are passing both `state` and `dispatch` wouldn't be more convenient to refactor this function to thunk function?  (`const getAccountNotification = () => (dispatch: Dispatch, getState: GetState) =>...`) @mroz22 @vladimirvolek what do you think?